src/pages/home/report/ReportActionCompose.js

```diff
@@ -230,6 +231,75 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Returns the list of report compose actions
+     *
+     * @param {Boolean} showSplitBill
+     * @param {Boolean} showSendRequestMoney
+     * @param {Boolean} hasExcludedIOUEmails
+     * @param {Function} openPicker
+     * @param {Function} displayFileInModal
+     * @returns {Array<object>}
+     */
+    getReportComposeActions(showSplitBill, showSendRequestMoney, hasExcludedIOUEmails, openPicker, displayFileInModal) {
+        return [
```

Comment:
Do we really need to compute and pass everything to this function it does not make any sense?

Reply:
Resolved.